### ISSUE TITLE
Not returning distance education courses for UOG

### ISSUE THREAD

**Issue by jacobmichels** [EvilKanoa/webadvisor-api#25]
Courses that are distance education format are not returned when searched for.

```
{
  search(query:"ECON",term:W20,institution:UOG)
  {
    results
    {
      code
    }
  }
}

```

For example, there is an ECON 1050 section that is distance education for the W20 semester and it does not show up from this search.

**Comment by RMKanoaHaley** [comment]
I cannot seem to reproduce this issue. Maybe it was a result of some bug on webadvisor's side of things?

As a quick demo, you can use the follow query to output all section codes for all W20 ECON courses:
```
{
  search(query:"ECON",term:W20,institution:UOG)
  {
    results
    {
      course {
        sections {
          id
        }
      }
    }
  }
}
```

If you execute that query, it should be noted that DE course sections do appear in that list.

**Comment by RMKanoaHaley** [comment]
I cannot seem to reproduce this issue. Maybe it was a result of some bug on webadvisor's side of things?

As a quick demo, you can use the follow query to output all section codes for all W20 ECON courses:
```
{
  search(query:"ECON",term:W20,institution:UOG)
  {
    results
    {
      course {
        sections {
          id
        }
      }
    }
  }
}
```

If you execute that query, it should be noted that DE course sections do appear in that list.

**Comment by EvilKanoa** [comment]
After some more investigation, I have determined that this problem does still exist. It does not seem to occur with all DE courses, but I have not seen it with non-DE courses which leads me to believe it is somehow related to DE courses. After some trial and error work, it appears that the original static POST params that were getting sent on every course lookup were causing **some** DE courses not to appear.

In particular, I have isolated the following params:
```
{
      'VAR10': 'Y',
      'VAR11': 'Y',
      'VAR12': 'Y',
      'VAR13': 'Y',
      'VAR14': 'Y',
      'VAR15': 'Y',
      'VAR16': 'Y',
}
```

If these params are removed from the requests sent to WebAdvisor, all DE courses seem to be returned. I'll open a PR with these changes.
I'm not convinced that this is the most eloquent of solutions or if it even fixes all issues, so I will be open to re-open this ticket if the issue re-surfaces or if we are able to come up with a more stable and reliable way to mimic the UI interactions that occur on WebAdvisor.
